oprish/src/routes/mod.rs

```diff
@@ -1,17 +1,21 @@
 pub mod messages;
-pub mod rate_limits;
 
 use rocket::{serde::json::Json, Route, State};
 use rocket_db_pools::Connection;
-use todel::{http::ClientIP, models::InstanceInfo, Conf};
+use todel::{
+    http::ClientIP,
+    models::{InstanceInfo, InstanceRateLimits},
+    Conf,
+};
 
 use crate::{
     rate_limit::{RateLimitedRouteResponse, RateLimiter},
     Cache, VERSION,
 }; // poggers
 
-#[get("/")]
+#[get("/?<rate_limits>")]
```

Comment:
Wrong base branch?

Reply:
Nay, that's just how you do query params in rocket!

Reply:
Nvm that was just me merging into wrong branch, my bad!

Reply:
See, wrong base branch.

Reply:
mmmmmmmmmmmmm